modoboa/admin/tests/test_need_dovecot_update.py

```diff
@@ -0,0 +1,86 @@
+"""Management command tests."""
+
+import os
+import shutil
+import tempfile
+from unittest import mock
+
+from django.core.management import call_command
+from django.test import override_settings
+from django.urls import reverse
+from django.db import transaction
+
+from modoboa.core.tests.test_views import SETTINGS_SAMPLE
+
+from modoboa.lib.tests import ModoTestCase
+from .. import factories, models
+
+from modoboa.core import models as core_models
+
+class NeedDovecotUpdateTestCase(ModoTestCase):
+    """Test need dovecot ldap update command."""
+
+    @classmethod
+    def setUpTestData(cls):
+        """Create test data."""
+        super(NeedDovecotUpdateTestCase, cls).setUpTestData()
+        factories.populate_database()
+
+    def setUp(self):
+        """Initiate initial env."""
+        super(NeedDovecotUpdateTestCase, self).setUp()
+        self.workdir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        """Reset test env."""
+        shutil.rmtree(self.workdir)
+
+    def test_update_dovecot_update_state_valid_form(self):
+        localconfig = core_models.LocalConfig.objects.first()
+        self.assertFalse(localconfig.need_dovecot_update)
+        url = reverse("core:parameters")
+        settings = SETTINGS_SAMPLE.copy()
+        response = self.client.post(url, settings, format="json")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.json(), "Parameters saved")
+        localconfig.refresh_from_db()
+        self.assertTrue(localconfig.need_dovecot_update)
+
+    def test_update_dovecot_ldap_conf(self):
+        localconfig = core_models.LocalConfig.objects.first()
```

Comment:
There is already a `localconfig` instance available. Use  `self.localconfig` instead and your test should pass.